src/signals/incident-management/components/FilterForm/index.js

```diff
@@ -146,18 +157,62 @@ const FilterForm = ({
     state.options.category_slug
   )
 
+  const sum = (previousValue, currentValue) => previousValue + currentValue
+
   const dateFrom =
     state.options.created_after && new Date(state.options.created_after)
   const dateBefore =
     state.options.created_before && new Date(state.options.created_before)
 
+  useEffect(() => {
+    /*
+     * The purpose is to calculate the length of the url because it might be too long.
+     * ParseOutputFormData returns all the checked filtersm in the following form:
+     * {status: ['g', 'i', ...], stadsdeel: ['w', ...], date_created_after: '11-11-1911'}
+     * Every key with each of its values is added to the URL as '&key=value', that's why we
+     * calculate for every key and each of its values the length of the key and of the value
+     * and sum that.
+     */
+    const filterLength =
```

Comment:
(options) => Object.entries(options).reduce((lengthOptions, [key, value]) => {
const allArray = Array.isArray(value) ? value : [value] // is dit niet altijd een array?
return lengthOptions + allArray.reduce((valueSum, s) => valueSum + key.length + 2 + s.length,0)
}, 57)

Reply:
met parseOutputFormData(state.options)